llvm/test/tools/llvm-objdump/ELF/AMDGPU/kd-gfx12.s

```diff
@@ -0,0 +1,54 @@
+;; Test disassembly for gfx12 kernel descriptor.
+
+; RUN: rm -rf %t && split-file %s %t && cd %t
+
+;--- 1.s
+; RUN: llvm-mc --triple=amdgcn-amd-amdhsa -filetype=obj -mcpu=gfx1200 < 1.s > 1.o
+; RUN: llvm-objdump --disassemble-symbols=kernel.kd 1.o | tail -n +7 | tee 1-disasm.s | FileCheck 1.s
```

Comment:
This is copied from `kd-gfx10.s` and other files that have been there for a while.

Reply:
It's using `tail` to chop some junk off the front of the output of `llvm-objdump --disassemble-symbols` so it can feed it back into `llvm-mc`. I could use `sed` instead if that seems more portable.